### PR TITLE
[65] Ignore bots. Update deps.

### LINKED ISSUE #65: Ignore commands from Codecov
Sometimes ([like here](https://github.com/chevah/server/pull/6693#issuecomment-2103771466)), chevah-robot reacts to a Codecov comment.

Ignore Codecov, its  messages are intended for people. Maybe briefly look at a more general way to tell if it's an automated comment.

### PR DESCRIPTION
**Pull request by danuker**
Fixes #65. Ignores comments from all bots.

The functionality change was easy enough; but I needed to install the Azure tools again, which was not easy.

I also updated the dependencies, as a by-product of recreating the environment. We'll see if it still works.
Reviewers: @adiroiban

### CODE REVIEW

**Comment by danuker** [comment]
Update: I noticed it wanted to upload 1.57 GB. I cancelled; the package should not be that big. I need to debug some more.

File: chevah/github_hooks_server/handler.py [hunk 1]
```diff
@@ -317,6 +317,9 @@ def issue_comment(self, event):
             action='issue_comment',
             )
 
+        if commenter_name.endswith('[bot]'):
```
Comment (adiroiban) [inline chevah/github_hooks_server/handler.py:320]: Hm... is this how github advertises the bot accounts?

I was expecting to have a separate attribute to signal that the user is a bot

image

image

Comment (danuker) [reply chevah/github_hooks_server/handler.py:320]: Yes, there is also a User "type" field which I should probably use, but also the login has a [bot] suffix in the API, and it was easier this way.

**Comment by danuker** [comment]
I don't know why I built such a big package, but now I have a safeguard against it, showing the size from `build-package.sh`. 
I have also deployed the commit [9b857b0](https://github.com/chevah/github-hooks-server/pull/66/commits/9b857b02fdd44251ba26c08f38e0513b95b960e4).
Let's also see if it still works: needs-review

**Comment by danuker** [comment]
Did a drive-by change of making the reviewers tag case-insensitive. needs-review

**Review by adiroiban — APPROVED** [review]
Looks good. thanks.

I have tested it here https://github.com/chevah/server/pull/6715#issuecomment-2130174132

**Pull request merged**